File: 392-is-subsequence/is-subsequence.py
class Solution(object):
    def isSubsequence(self, s, t):
        if len(s)>len(t):
            return False
        one,sec=0,0
        flag=1
       
        while one<len(s) and sec<len(t):
            if(s[one]==t[sec] ):
                one+=1
                sec+=1
                if(one==len(s)):
                    flag=1
            else:
                sec+=1
                flag=0
        return flag
        

File: 392-is-subsequence/is-subsequence.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isSubsequence = function(s, t) {
    let l=0;
    let r=0;
    
    while(r<t.length){
        if(s[l]==t[r]){
            l++;
            r++;
        }
        else{
            r++;
        }
    }
    if(l==s.length){
        return true;
    }
    else{
        return false;
    }
    
};